rest/config_test.go

```diff
@@ -264,8 +265,8 @@ func TestConfigValidationImportPartitions(t *testing.T) {
 			assert.NoError(tt, err)
 			errorMessages := config.setupAndValidateDatabases()
 			if test.err != "" {
-				require.Len(tt, errorMessages, 1)
-				assert.EqualError(tt, errorMessages[0], test.err)
+				require.NotNil(tt, errorMessages)
+				assert.EqualError(tt, errorMessages.Errors[0], test.err)
```

Comment:
length check first

Reply:
Sure, this is addressed in latest revision.